src/staking-v3/logic/interfaces/DappStakingV3.ts

```diff
@@ -127,13 +127,12 @@ export interface PalletDappStakingV3ContractStakeAmount extends Struct {
 }
 
 export interface PalletDappStakingV3TiersConfiguration extends Struct {
```

Comment:
```suggestion
export interface PalletDappStakingV3TiersConfigurationLegacy extends Struct {

export interface PalletDappStakingV3TiersConfiguration extends Struct {
```

@bobo-k2 you need 2 different versions here and try to decode with new one and fallback to legacy one. Otherwise this will fail until we perform the migration

Reply:
I tested new version on local node and on Astar. Decoding works, but I will check how to improve

Reply:
@bobo-k2 It will decode because you're telling the type but the values will be incorrect.

Reply:
You are right. Fixed in this [commit](https://github.com/AstarNetwork/astar-apps/pull/1376/commits/24af84ebdc616ada5066928236eec7a9a016f085)